page_layouts/product.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Products | GreenAlley</title>
+  <script src="https://unpkg.com/phosphor-icons"></script>
+  <link rel="stylesheet" href="https://fab-ui-lib.netlify.app/stylesheets/main.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" />
+  <link rel="stylesheet" href="/stylesheets/style.css" />
+  <link rel="stylesheet" href="/stylesheets/page/product.css" />
+</head>
+
+<body>
+  <!-- navbar -->
+  <nav class="gh-navbar nav-wrap nav-search-wrap">
+    <div class="gh-nav-head nav-head">GreenAlley</div>
+
+    <div class="gh-search-holder nav-search-holder-icon">
+      <i class="fa-solid fa-seedling nav-search-icon fas"></i>
+      <input type="search" class="input gh-search" placeholder="Search for products, brands and more" />
+    </div>
+
+    <div class="nav-icon-wrap">
+      <div class="nav-icon-chip">
+        <i class="fas fa-user"></i>
+        <span class="nav-icon-text">Log In</span>
+      </div>
+      <div class="nav-icon-chip">
+        <i class="fas fa-heart"></i>
+        <span class="nav-icon-text">Wishlist</span>
+      </div>
+      <div class="nav-icon-chip">
+        <i class="fas fa-cart-arrow-down"></i>
+        <span class="nav-icon-text">Cart</span>
+      </div>
+    </div>
+  </nav>
+  <!-- navbar ends -->
+
+  <!-- main content -->
+
+  <div class="gh-main-wrap">
+
+    <!-- filter -->
+    <aside class="gh-filter-wrap">
+      <div class="gh-filter-wrap-row">
+        <div class="h6 text-bold gh-filter-title">Filters</div>
+        <div class="h6 gh-filter-clear gh-filter-title">Clear</div>
+      </div>
+      <div class="gh-filter-wrap-col gh-filter-margin">
+        <div class="h5 text-bold gh-filter-title">Price</div>
+        <div class="gh-filter-slider">
+          <span class="gh-slider-text">0</span>
+          <span class="gh-slider-text">500</span>
+          <span class="gh-slider-text">5000</span>
+        </div>
+        <input type="range" name="" id="" class="gh-slider">
```

Comment:
suggestion not to keep fields blank. Remove it entirely if you don't need it.

Reply:
done as suggested.